internal/command/launch/state.go

```diff
@@ -75,7 +75,7 @@ func (state *launchState) Region(ctx context.Context) (api.Region, error) {
 		return r.Code == state.Plan.RegionCode
 	})
 	if !ok {
-		return region, fmt.Errorf("region %state not found", state.Plan.RegionCode)
+		return region, fmt.Errorf("region %s not found", state.Plan.RegionCode)
```

Comment:
Or even **(except that `maa` _does_ show up in `fly platform regions` output right now)**:

```suggestion
		return region, fmt.Errorf("region %s not found. Is %s a valid region according to 'fly platform regions'?", state.Plan.RegionCode)
```

(I'm not sure how we indicate commands in flyctl messages; I'm pretty sure I've seen single quotes used, though.)